docs/index.md

```diff
@@ -5,49 +5,4 @@ feature_maturity: stable
 enterprise: 'no'
 ---
 
-Apache Spark is a fast and general-purpose cluster computing system for big data. It provides high-level APIs in Scala, Java, Python, and R, and an optimized engine that supports general computation graphs for data analysis. It also supports a rich set of higher-level tools including Spark SQL for SQL and DataFrames, MLlib for machine learning, GraphX for graph processing, and Spark Streaming for stream processing. For more information, see the [Apache Spark documentation][1].
```

Comment:
i can move back in to this project. for the dcos-docs-enterprise content i can add a simple landing page header, similar to `Welcome to the documentation for Apache DC/OS Spark. Choose a version at the left to get started!` - and leave the rest of the content in this repo.